### PR TITLE
chore(release): add libraries to release page

### PR DESCRIPTION
**Pull request by jeqo**
Tested on a fork, looks like this: https://github.com/jeqo/tiered-storage-for-apache-kafka/releases/tag/v0.0.1

### CODE REVIEW

File: .github/workflows/create_release.yml [hunk 1]
```diff
@@ -74,22 +96,75 @@ jobs:
           draft: true
           prerelease: false
 
-      - name: Upload tar
+      - name: Upload core tar
         uses: actions/upload-release-asset@v1
```
Comment (AnatolyPopov) [inline .github/workflows/create_release.yml:100]: This action is unmaintained and I guess it's a good time to switch to a more recent variant of this.

**Review by AnatolyPopov — COMMENTED** [review]
I see that in the example you provided the name of the `core` artifact is `tiered-storage-for-apache-kafka` that is confusing IMO. I would suspect artifact with such name to contain a complete distro, however core will not work without storage implementation. WDYT?

**Comment by jeqo** [comment]
@AnatolyPopov thanks! Yes, agree on the core libs naming. I have changed that to pick the libs from core/build instead. See the last commit and release sample: https://github.com/jeqo/tiered-storage-for-apache-kafka/releases/tag/v0.0.1

**Review by AnatolyPopov — APPROVED** [review]
LGTM

**Review by ivanyu — APPROVED** [review]

**Pull request merged**